tools/genevergreen/main.go

```diff
@@ -60,11 +76,153 @@ func buildDependency(toolName, os, serverVersion, repo string) shrub.TaskDepende
 		"debian10":     "debian10",
 		"debian11":     "debian11",
 	}
+)
 
+func buildDependency(toolName, os, serverVersion, repo string) shrub.TaskDependency {
 	return shrub.TaskDependency{
-		Name:    fmt.Sprintf("push_%v_%v_%v_stable", toolName, newOs[os], repo),
-		Variant: fmt.Sprintf("release_%v_publish_%v", toolName, strings.ReplaceAll(serverVersion, ".", "")),
+		Name:    fmt.Sprintf("push_%s_%s_%s_stable", toolName, newOs[os], repo),
+		Variant: fmt.Sprintf("release_%s_publish_%s", toolName, strings.ReplaceAll(serverVersion, ".", "")),
+	}
+}
+
+type Platform struct {
+	extension     string
+	architectures []string
+}
+
+// if updating this list verify build/ci/repo_config.yaml matches.
+var distros = map[string]Platform{
```

Comment:
nit: move all vars together

Reply:
going to merge and if I don't need to revert I'll do a follow up